utils.py

```diff
@@ -389,15 +389,24 @@ def getTimeDiffFormatted(time_1 = None,
     time_obj -- if simply just want to convert a time object (time delta) into a CSV-friendly string, pass it to this argument
     """
 
+    if time_1 is None and time_2 is None and time_obj is None:
+        return None, None
+
     if time_1 and time_2:
-        time_delta = time_2 - time_1
+        try:
+            time_delta = time_2 - time_1
+        except TypeError as e:
+            print(f"Error: {e}")
+            return None, None
     elif time_obj:
         time_delta = time_obj
     else:
         return None, None
-    if time_delta.days != 0:  
+
+    if time_delta.days != 0:
         d = {"days": time_delta.days}
-        d["hours"], rem = divmod(time_delta.seconds, 3600)
+        hours, rem = divmod(time_delta.seconds, 3600)
+        d["hours"] = hours
```

Comment:
what's the point of this change?

Reply:
I think here it's just easier to understand

Reply:
It's  a pointless change and unnecessarily creates a new variable while not really making a real change as the values are still assigned the same way but not with extra steps

Reply:
ok